community/fulltext-index/src/main/java/org/neo4j/kernel/api/impl/fulltext/LuceneFulltextDocumentStructure.java

```diff
@@ -85,6 +172,7 @@ static Term newTermForChangeOrRemove( long id )
         return new Term( FIELD_ENTITY_ID, "" + id );
     }
 
+    // todo Does this need to be adjusted for the number search support?
```

Comment:
Is this todo something that should be addressed before merging?

Reply:
I actually had addressed it, I just need to take out the `todo` I'll do that